Providers/Pocket/PocketFeed.swift

```diff
@@ -76,24 +36,33 @@ class Pocket: FeatureFlaggable, URLCaching {
     }()
 
     // Fetch items from the global pocket feed
-    func globalFeed(items: Int = 2) -> Deferred<Array<PocketStory>> {
+    func globalFeed(items: Int = 2) -> Deferred<[PocketFeedStory]> {
```

Comment:
Just a note, if possible we should move away from deferred

Reply:
Sounds good to me. I was keeping it for consistency. Is Result type ok?

Reply:
There's plan to use Combine in the future (no puns intended lol) in the app, but we don't use it yet. Result type is used in other places in the app so would work well too.

Reply:
Next PR containing API integration will use `Result` type.